build.gradle

```diff
@@ -39,7 +39,7 @@ sourceSets {
 
 // Remove all .proto definition from the final build
 processResources {
-    exclude('**/*')
+    exclude('POGOProtos/')
```

Comment:
next time create a second pr for this feature. But it should be ok now to merge them together.